backend/src/payments/payment.dto.ts

```diff
@@ -13,6 +13,10 @@ export class PaymentsDto {
   @ApiModelProperty()
   readonly payment_status: boolean;
 
+  @IsString()
```

Comment:
Shouldnt this be `IsBoolean()` ? same for `payment_status` that is existing wrong code.